packages/core/src/testInteg/perf/collectFiles.test.ts

```diff
@@ -8,55 +8,89 @@ import * as sinon from 'sinon'
 import { performanceTest } from '../../shared/performance/performance'
 import { createTestWorkspaceFolder, toFile } from '../../test/testUtil'
 import path from 'path'
-import { randomUUID } from '../../shared'
+import { fs, randomUUID } from '../../shared'
 import { collectFiles } from '../../shared/utilities/workspaceUtils'
+import { getFsCallsUpperBound } from './utilities'
+import { FileSystem } from '../../shared/fs/fs'
 
-performanceTest(
-    // collecting all files in the workspace and zipping them is pretty resource intensive
-    {
-        linux: {
-            userCpuUsage: 85,
-            heapTotal: 2,
-            duration: 0.8,
+function performanceTestWrapper(totalFiles: number) {
+    return performanceTest(
+        {
+            darwin: {
+                userCpuUsage: 100,
```

Comment:
it looks like a lot of these values are just declared once and then used in darwin/linux/win32. Does it make sense to just have a field called default or something that can be passed in and used as the default for everything?

Reply:
Yeah, now that the thresholds are looser I think it makes sense to factor this out to reduce clutter. I wrote a helper function to reduce this.